jib-core/src/main/java/com/google/cloud/tools/jib/frontend/BuildDockerStepsRunner.java

```diff
@@ -190,4 +193,20 @@ private void handleRegistryUnauthorizedException(
           registryUnauthorizedException);
     }
   }
+
+  /** @return true if Docker is installed on the user's system. */
+  public static boolean isDockerInstalled() {
+    try {
+      ProcessBuilder processBuilder = new ProcessBuilder("docker");
+      processBuilder.redirectErrorStream(true);
+      Process process = processBuilder.start();
+      try (InputStreamReader stdout =
```

Comment:
Is this stdout reading necessary? Could it just be start the process, if exception then return false, otherwise true?

Reply:
Looks like that works.